server/services/work_experience.js

```diff
@@ -0,0 +1,22 @@
+const { Pool } = require("pg");
+const config = require("../../config");
+const pool = new Pool(config);
+
+const createWorkExperience = (experience) => {
+  return pool
+    .query(
+      "INSERT INTO work_experience (company, job_title, description, start_date, end_date) values ($1, $2, $3, $4, $5)",
+      [
+        experience.company,
+        experience.job_title,
+        experience.description,
+        experience.start_date,
+        experience.end_date,
+      ]
+    )
+    .then((result) => result.rows);
+};
+
+module.exports = {
+  createWorkExperience,
+};
```

Comment:
I do not think we need this file, you already created it on `server/services/database/work_experience.js`